create-custom-map-style/README.md

```diff
@@ -0,0 +1,471 @@
+
+With [Amazon Location Service](https://aws.amazon.com/location/), a managed AWS service for adding location data to applications, you can create a customized web map that will grab the interest of your end users and increase user engagement. At this time Location Service has two data providers, [Esri](https://aws.amazon.com/location/data-providers/esri/) and [HERE](https://aws.amazon.com/location/data-providers/here-technologies/), as well as six default map styles. So, how do you customize these map styles to meet your brand or visual design?
+
+This README will walk you through the process of styling an existing map using [Maputnik](https://maputnik.github.io/) to help visualize the changes in real-time. Others may prefer to use another open source tool called [Fresco](https://fresco.go-spatial.org/), however they are not required to make edits to the style descriptor. In order to style our existing map from Amazon Location Service using these tools we will also demonstrate how to run a local proxy that handles [Signature Version 4 / AWS Authentication](https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html).
+
+The following walk through is split into 2 parts:
+
+1) [How to style an existing Map from Amazon Location Service section](#how-to-style-an-existing-map-from-amazon-location-service).
+2) [How to integrate a custom map style in a React app](#how-to-integrate-a-custom-map-style-in-a-react-app)
+
+## Prerequisites
+
+1. [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+
+1. [NVM](https://github.com/nvm-sh/nvm#installing-and-updating) (installed per-user, and invoked per-shell) OR [Node.js](https://nodejs.org/en/download/)
+
+1. [AWS-CLI](https://docs.amplify.aws/lib/geo/getting-started/q/platform/js/#provisioning-resources-through-cli)
+
+1. [Amplify CLI](https://docs.amplify.aws/cli/start/install) (Note: If you have a local environment that uses several AWS accounts be sure to use the correct AWS CLI profile and log in to the correct account in the browser)
+
+1. This walk-through uses **"Amplify-Default"** as the AWS CLI profile name and **"us-west-2"** as the AWS Region. Feel free to substitute these values.
+
+## How to style an existing Map from Amazon Location Service
+
+1. Maputnik is an open source visual editor for the [Mapbox Style Specification](https://www.mapbox.com/mapbox-gl-js/style-spec). At the time of writing this editor can be either used in the browser or downloaded as a binary and can run locally. We are going to use the latter option.
+
+1. Go to <https://github.com/maputnik/editor/wiki/Maputnik-CLI> and follow the instructions for installation or optionally from your terminal you can run:
+
+    ```bash
+
+    wget https://github.com/maputnik/editor/releases/download/v1.7.0/maputnik-darwin.zip
```

Comment:
Re-highlighting that this is Mac-specific and should probably be acknowledged up-front / nearby.